Patches/PlayerControlPatch.cs

```diff
@@ -831,7 +831,7 @@ public static void AfterReportTasks(PlayerControl player, GameData.PlayerInfo ta
 
         foreach (var pc in Main.AllPlayerControls)
         {
-            if (!Doppelganger.CheckDoppelVictim(pc.PlayerId))
+            if (!Doppelganger.CheckDoppelVictim(pc.PlayerId) && !Murderer.CheckMurdererVictim(pc.PlayerId))
             {
                 if (target != null && Murderer.CheckMurdererVictim(target.PlayerId)) return;
```

Comment:
btw I also meant you should remove this, because this skips the entire code beneath it when it's true, and that should not happen.